README.md

```diff
@@ -80,35 +102,69 @@ This section explains how to register the application as a single tenant applica
+To do this replacement in the manifest, you have two options:
+    - Option 1: Edit the manifest in place by clicking **Edit**, replacing the appRoles value, and then clicking **Save**. 
+    - Option 2: **Download** the manifest to your computer, edit it with your favorite text editor, save a copy of it, and **Upload** this copy. You might want to choose this option if you want to keep track of the history of the manifest.
+
+The content of `appRoles` should be the following (the `ìd` can be any unique GUID)
 ```JSON
 "appRoles": [
     {
       "allowedMemberTypes": [
         "User"
       ],
-      "description": "Admins can manage roles and perform all task actions.",
+      "displayName": "Writer",
+      "id": "d1c2ade8-98f8-45fd-aa4a-6d06b947c66f",
+      "isEnabled": true,
+      "description": "Writers Have the ability to create tasks.",
+      "value": "Writer"
+    },
+    {
+      "allowedMemberTypes": [
+        "User"
+      ],
+      "displayName": "Observer",
+      "id": "fcac0bdb-e45d-4cfc-9733-fbea156da358",
+      "isEnabled": true,
+      "description": "Observers only have the ability to view tasks and their statuses.",
+      "value": "Observer"
+    },
+    {
+      "allowedMemberTypes": [
+        "User"
+      ],
+      "displayName": "Approver",
+      "id": "fc803414-3c61-4ebc-a5e5-cd1675c14bbb",
+      "isEnabled": true,
+      "description": "Approvers have the ability to change the status of tasks.",
+      "value": "Approver"
+    },
+    {
+      "allowedMemberTypes": [
+        "User"
+      ],
       "displayName": "Admin",
       "id": "81e10148-16a8-432a-b86d-ef620c3e48ef",
       "isEnabled": true,
-      "origin": "Application",
+      "description": "Admins can manage roles and perform all task actions.",
       "value": "Admin"
-    },
-    <<<< Approver, Observer, & Writer roles go here >>>>
+    }
   ],
 ```
 
 ### Step 3:  Configure the sample to use your Azure AD tenant
 
-1. Open the solution in Visual Studio 2013.
+1. Open the solution in Visual Studio.
 2. Open the `web.config` file.
 4. Find the app key `ida:ClientId` and replace the value with the Application ID for the application from the Azure portal.
 5. Find the app key `ida:AppKey` and replace the value with the key for the application from the Azure portal.
 6. Find the app key `ida:Tenant` and replace the value with the domain of your tenant.
 6. If you changed the base URL of the TodoListWebApp sample, find the app key `ida:PostLogoutRedirectUri` and replace the value with the new base URL of the sample.
-7. In `Startup.Auth.cs`, comment out or delete the lines corresponding to the multi-tenant version of the sample, which are marked by comments.  You'll have to change the value for the `Authority` to the single-tenant version, and delete the line relating to `ValidateIssuer` in `TokenValidationParameters`.
+7. In `Startup.Auth.cs`, comment out the first line: `// #define SingleTenantApp`. 

```

Comment:
Providing guidance to leverage the updated Startup.Auth.cs file where I've introduced conditional compilation (#if SingleTenantApp), so that things are easier for the user and less error prone.